source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py

```diff
@@ -274,6 +275,11 @@ def get_places(self, obj):
             if not type_ or type_ not in map_serializer:
                 logging.getLogger(__name__).error('Place not serialized (unknown type): {}'.format(feature))
                 continue
+            zone_type = feature.get('properties', {}).get('geocoding', {}).get('zone_type')
```

Comment:
Would be nice to read the geocoding object from the feature only once (see line 274 above).